### DISCUSSION TITLE
AddJaegerExporter throws if no host is available

### DISCUSSION THREAD

**Discussion by ImoutoChan** [open-telemetry/opentelemetry-dotnet#5655] [Q&A]
How to disable this behavior? Tracing is not critical for the application and shouldn't crash app.

**Comment by cijothomas** [comment]
`JaegerExporter` is deprecated and removed from the repo, so there won't any changes/fixes to `JaegerExporter`. OTLPExporter is the recommended alternative for exporting to Jaeger, and OTLPExporter does not crash/throw if the endpoint is not available.
